packages/runtime/container-runtime/src/inboundBatchAggregator.ts

```diff
@@ -82,55 +40,30 @@ class ScheduleManagerCore {
 		private readonly getClientId: () => string | undefined,
 		private readonly logger: ITelemetryLoggerExt,
 	) {
```

Comment:
Maybe paranoid, feel free to veto - but consider keeping `prepareSend` handler and have it assert that the batch metadata is correct.

Reply:
I think the argument why it's not necessary is that since the BatchManager is also in runtime layer, there's no back compat here - this code should have been removed long ago, in the same PR that updated BatchManager to set start and end batch metadata.

Reply:
Yep, they are in the same layer so there is no back-compat concern.